### DISCUSSION TITLE
Dark Theme

### DISCUSSION THREAD

**Discussion by ghost** [rodit/SnapMod#111] [Ideas & Feature Suggestions]
Dark theme support like there is on IOS. Or somehow make it more compatible with DarQ's force dark?
Love this project btw!

**Comment by rodit** [comment]
There's no easy way to get dark theme to work on Snapchat unfortunately - Snapchat seems to be in the process of moving their UI towards (or away from, it's hard to tell) their 'composer' framework which seems to have dark mode built in but toggling it simply makes about 20% of certain screens of the app black (it looks awful).

**Reply by authorisation** [reply]
What exactly would someone have to do to implement dark mode correctly?

**Comment by RevealedSoulEven** [comment]
Actually you can use System UI dark mode for snapchat but it's very annoying. I mean dark mode is not actually meant for apps like Snapchat as it completely hurts the UI of the app. Besides, it's better on the original UI too and if you start using Dark Mode then you'll leave it within few minutes.

And it's a concern for the developers to think about it

**Comment by hockeymikey** [comment]
I could never get dark mode working with any theme in substratum or swift installer. I would like dark mode, less blinding at night. Would me being on Oreo be a factor for this? I wouldn't think it would effect an overlay, but is it something to do with the new UI framework mentioned above?

**Reply by ghost** [reply]
You can use DarQ but it is far from perfect.

**Reply by hockeymikey** [reply]
@AuzGitHub But DarQ is only 10 and above not 8.

**Reply by tycoon420** [reply]
Use Hex Installer : OneUi theme if you are using samsung device. 
![Screenshot_20221023-180056_Snapchat](https://user-images.githubusercontent.com/11652113/197392376-a51d5280-3d2b-42dd-b1dc-e572ead505a1.jpg)
![Screenshot_20221023-180109_Snapchat](https://user-images.githubusercontent.com/11652113/197392388-9aeb8441-e9a2-4fe6-bd54-5323726c4987.jpg)

**Reply by hockeymikey** [reply]
I do not have a Samsung @tycoon420

**Reply by tycoon420** [reply]
> I do not have a Samsung @tycoon420

Use Andromeda and Swiftblack

**Reply by ozilwg** [reply]
On samsung android 13 one ui 4.1 is it possible to activate the dark mode somehow?  Without the use of hex installer
